### ISSUE TITLE
Move non essential plugin information into wiki pages

### ISSUE THREAD

**Issue by hferentschik** [projectatomic/adb-vagrant-registration#71]
The _README_ is quite long and contains information which is not directly relevant for the user or developer. It should be shortened and information which is still valuable, but not suitable for the README should be moved to for example wiki pages.

**Comment by pvalena** [comment]
Yes, the think the Usage section could be split into some basic usage, which would stay in README, and extended usage, which would be in a separate document (or wiki page).
